ndcube/extra_coords/extra_coords.py

```diff
@@ -298,10 +312,13 @@ def _getitem_lookup_tables(self, item):
         for lut_axis, lut in self._lookup_tables:
             lut_axes = (lut_axis,) if not isinstance(lut_axis, tuple) else lut_axis
             lut_slice = tuple(item[i] for i in lut_axes) if isinstance(item, tuple) else item
+            if isinstance(lut_slice, tuple) and len(lut_slice) == 1:
+                lut_slice = lut_slice[0]
 
             sliced_lut = lut[lut_slice]
-            if sliced_lut:
+            if not isinstance(lut_slice, Integral):
                 new_lookup_tables.add((lut_axis, sliced_lut))
+            # TODO: Handle dropped table here
```

Comment:
Is this to be fixed in this PR or is this where a new PR should address #349?

Reply:
Probably a follow on one.